EMULATORS.md

```diff
@@ -55,3 +55,4 @@
 | [Y2Dmg](https://github.com/Y2JB/DMG) | C# | Fully featured DMG emulator written in C#. Passes Blargg's test roms and supports most games. Core is cross platform (.net core), renderer is Windows only.|
 | [Gandalf](https://github.com/stan-roelofs/Gandalf) | C++ | Cross-platform GB, GBC emulator with an SDL/ImGui frontend that provides several debugging features. |
 | [Boytacean](https://github.com/joamag/boytacean) | Rust | A Game Boy emulator that is written in Rust, with a beautiful Web interface that makes use of WASM. |
+| [Gameboy](https://github.com/raphamorim/gameboy) | Rust | Gameboy emulator written in Rust to WebAssembly that runs as a Desktop or Web application |
```

Comment:
I would say written in "Rust to WebAssembly" is a bit confusing...what about "Runs in the browser through Webassembly"? Check also other entries in the list mentioning webassembly.

Reply:
good point, updated what you think?